src/test/javascript/spec/service/faq.service.spec.ts

```
@@ -0,0 +1,209 @@
+            req.flush(returnedFromService);
+            expect(req.request.method).toBe('DELETE');
+        });
+
+        it('should find a faq', () => {
+            const category = {
+                color: '#6ae8ac',
+                category: 'category1',
+            } as FAQCategory;
+            const returnedFromService = { ...elemDefault, categories: [JSON.stringify(category)] };
+            const expected = { ...elemDefault, categories: [new FAQCategory('category1', '#6ae8ac')] };
+            const faqId = elemDefault.id!;
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)